### PR TITLE
feat(runInstalledQuery and runAlgorithm): add memory limit header

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/706/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/707/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/708/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/709/

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/710/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

File: pyTigerGraph/gds/featurizer.py [hunk 1]
```diff
@@ -507,6 +508,9 @@ def runAlgorithm(
             threadLimit:
                 Specify a limit of the number of threads the query is allowed to use on each node of the TigerGraph cluster.
                 See xref:tigergraph-server:API:built-in-endpoints#_specify_thread_limit[Thread limit]
+            memoryLimit:
```
[the remark this answers is not in the document]
Comment (parkererickson-tg) [reply pyTigerGraph/gds/featurizer.py:511]: Added.

**Pull request merged**